xcube/server/api.py

```diff
@@ -48,9 +49,10 @@
     Dict[str, "JSON"],
 ]
 
-ServerConfig = Mapping[str, Any]
+ServerConfigObject = Mapping[str, Any]
+ServerConfig = ServerConfigObject
```

Comment:
I don't understand why this is necessary, but you introduced it for a reason, I'm sure.

Reply:
Yes, this significantly improves the readability of the code when used in type annotations. It adds semantics to arguments and return types. Common practice in Python. Btw, in generated API docs, the type aliases will go away.